api/tests/conftest.py

```diff
@@ -0,0 +1,90 @@
+from os import environ, path
+
+import boto3
+import pytest
+import yaml
+from moto import mock_dynamodb2
+
+from hyp3_api import DYNAMODB_RESOURCE, auth, connexion_app  # noqa hyp3 must be imported here
+
+AUTH_COOKIE = 'asf-urs'
+JOBS_URI = '/jobs'
+
+DEFAULT_JOB_ID = 'myJobId'
+DEFAULT_USERNAME = 'test_username'
+
+
+@pytest.fixture
+def client():
+    with connexion_app.app.test_client() as test_client:
+        yield test_client
+
+
+@pytest.fixture()
+def table():
+    table_properties = get_table_properties_from_template()
+    with mock_dynamodb2():
+        boto3.setup_default_session()
```

Comment:
I think this can go, along with `import boto3`? docs suggest calling it without parameters is the default behavior anyway